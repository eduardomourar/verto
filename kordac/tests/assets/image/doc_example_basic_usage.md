{image file_path="http://placehold.it/350x150" caption="Placeholder image" source="https://placehold.it/" title="This is hover text" alignment="left"}
