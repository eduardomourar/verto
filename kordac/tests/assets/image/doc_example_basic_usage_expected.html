<div>
  <img src="http://placehold.it/350x150" title="This is hover text" class="float-left"/>
  <p>Placeholder image</p>
  <p><a href="https://placehold.it/">Source</a></p>
</div>
